examples/templates/angular/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "angular",
```

Others templates are named like that:
```suggestion
  "name": "angular-template",
```